submissions/EnF1NeeT/HTML-CSS_popup/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```

Comment:
Don't forget about the mobile version, right now it's hard to read page content.
![image](https://user-images.githubusercontent.com/16196199/185800725-ff33e62e-65d0-43b7-86fb-9fc33193632b.png)

Reply:
Fixed in the last commit